Patch 7.1 (SCSS)

d3nik opened the pull request:
– some classes in html structure changed;
– fixed name-classes in css;
– created scss folder;
– exported css into scss compiler;
– main.scss consists of divided on modules scss files; – fixed css variables to scss variables;

d3nik committed:
Patch 7.1 (SCSS)

– some classes in html structure changed;
– fixed name-classes in css;
– created scss folder;
– exported css into scss compiler;
– main.scss consists of divided on modules scss files;
– fixed css variables to scss variables;

vozniuk-lntu reviewed (approved):
1. підправ на головній заливка для блоку з іконками в features
2. заголовки секцій збились
3. збилась зміна кольору при наведенні на ссилки в хедері

d3nik committed:
Patch 7.1.1 (hotfix)

– fixed header links, when they didn`t change a color;
– fixed captions to sections;
– fixed features icons background color;
